src/as-component-private.h

```diff
@@ -54,6 +54,9 @@ void			as_component_complete (AsComponent *cpt,
 						gchar *scr_base_url,
 						GPtrArray *icon_paths);
 
+gint			as_component_releases_sort_cb (gconstpointer a,
```

Comment:
This stuff is private API, we don't even export this function's symbol (everything in `*-private.h` headers is not exported, unless explicitly marked as internal API with `AS_INTERNAL_VISIBLE`.
However, this stuff tends to change, and it's nice to have a proper name in case we ever make this API public, so I changed the name of this function :-)